traffic_portal/app/src/common/api/StaticDnsEntryService.js

```diff
@@ -17,13 +17,69 @@
  * under the License.
  */
 
-var StaticDnsEntryService = function(Restangular) {
+var StaticDnsEntryService = function($http, $q, Restangular, locationUtils, messageModel, ENV) {
 
 	this.getStaticDnsEntries = function(queryParams) {
 		return Restangular.all('staticdnsentries').getList(queryParams);
 	};
 
+	this.getStaticDnsEntry = function(id) {
```

Comment:
technically you don't need this 2nd method but it won't hurt. because you can call getStaticDnsEntries like this: getStaticDnsEntries( { id: 4 }) and it will convert it to GET staticdnsentries?id=4

Reply:
I added the 2nd method so that a single SDE would be returned rather than an array containing a single element, it was a more straightforward response to handle in the EditController.

Reply:
k